public_html/wp-content/mu-plugins/tests/bootstrap.php

```diff
@@ -27,7 +27,7 @@ function manually_load_plugins() {
 
 	require_once dirname( __DIR__ ) . '/0-error-handling.php';
 	require_once dirname( __DIR__ ) . '/lets-encrypt-helper.php';
-	require_once dirname( __DIR__ ) . '/seo.php';
+	require_once dirname( __DIR__ ) . '/latest-site-hints.php';
```

Comment:
It looks like the [tests are failing](https://github.com/WordPress/wordcamp.org/runs/4746639950?check_suite_focus=true) because the cases need to be updated to reflect the changes in `get_latest_site_id()`. 

`test:watch -- --group=latest-site-hints` makes it convenient. [The Docker readme](https://github.com/WordPress/wordcamp.org/blob/f58e6fc89a5cb755574576b5e0ab935c57449e51/.docker/readme.md) has more details.